src/js/jobs/components/Item/Status.js

```diff
@@ -26,20 +26,30 @@ export const JobStatus = ({ pad, state }) => {
             </StyledJobStatus>
         );
     }
+    {
+        if (state === "complete") {
+            return (
+                <StyledJobStatus pad={pad}>
+                    <Icon name="check fa-fw" color="green" />
+                    <span>{state}</span>
+                </StyledJobStatus>
+            );
+        }
+    }
```

Comment:
```suggestion
        if (state === "complete") {
            return (
                <StyledJobStatus pad={pad}>
                    <Icon name="check fa-fw" color="green" />
                    <span>{state}</span>
                </StyledJobStatus>
            );
        }
```
Remove extra brackets